operator/cmd/operator/main.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"time"
+	"flag"
+	"runtime/pprof"
+
+	"istio.io/istio/operator/cmd/operator/cmd"
+	"istio.io/istio/operator/cmd/shared"
+)
+
+var cpuprofile = flag.String("cpuprofile", "", "write cpu profile to file")
```

Comment:
should not be in the production code, consider create {blah}_test.go and move all testing code there.

Reply:
agreed.